src/scripts/control.sh

```diff
@@ -0,0 +1,168 @@
+#!/bin/bash
```

Comment:
Where is this file used? I don't see any references to it.

Reply:
`src/descriptor/service.sdl`